pyaccel/lifetime.py

```diff
@@ -187,9 +301,9 @@ def accepen(self):
             return self._accepen
         dic = dict()
         rf_accep = self._eqpar.rf_acceptance
-        dic['spos'] = self._eqpar.twiss.spos
-        dic['accp'] = dic['spos']*0 + rf_accep
-        dic['accn'] = dic['spos']*0 - rf_accep
+        dic['spos'] = self._optics_data.spos
+        dic['accp'] = _np.minimum(self._accepen_pos, +rf_accep)
```

Comment:
I think it is better not to do this kind of operation under the hood like this.

Reply:
Ok, operation removed.